Source/Core/Core/Boot/Boot.h

```diff
@@ -42,7 +43,8 @@ class CBoot
                           std::string* title_id = nullptr);
 
 private:
-  static bool DVDRead(u64 dvd_offset, u32 output_address, u32 length, bool decrypt);
+  static bool DVDRead(const DiscIO::IVolume& volume, u64 dvd_offset, u32 output_address, u32 length,
```

Comment:
Note that by making this one by reference, and the others all by pointer, it makes for more visual noise at the point of usage.